domain-models-maven-plugin/src/test/resources/clients/TestClient.txt

```diff
@@ -134,14 +148,27 @@ public class TestResourceClient {
         if (okapiUrl!= null) {
             request.putHeader("X-Okapi-Url", okapiUrl);
         }
-        request.send(responseHandler);
+        return request.send();
     }
 
     /**
-     * Service endpoint "unittestsbooks"+queryParams.toString()
-     * 
+     * Service endpoint unittestsbooks
+     * @deprecated Use method, that returns Future
+     *
      */
+    @Deprecated
```

Comment:
Neither Vert.x nor RMB deprecate `Handler<AsyncResult<>>` methods. We continue to support them.

When writing new code or changing existing code developers will likely use futurised methods for ease of use.

Reply:
I removed deprication